krux/cli.py

```diff
@@ -96,6 +96,8 @@ def __init__(self, name, parser=None, logger=None, lockfile=False):
 
         :keyword parser: The CLI parser. Defaults to
         :py:func:`cli.get_parser <krux.cli.get_parser>`
+
+        :keyword cli_args: The cli arguments. when it = '', no any argument will be passed to CLI application.
```

Comment:
I don't like this comment either, but for a different reason. I would prefer a comment similar to [this one](https://github.com/krux/python-krux-stdlib/blob/release/krux/tornado/application.py#L109).